src/sentry/api/endpoints/auth_index.py

```diff
@@ -100,8 +112,21 @@ def put(self, request: Request):
             return Response(status=status.HTTP_401_UNAUTHORIZED)
 
         validator = AuthVerifyValidator(data=request.data)
-        if not validator.is_valid():
-            return self.respond(validator.errors, status=status.HTTP_400_BAD_REQUEST)
+        has_valid_sso_session = has_completed_sso(request, request.superuser.org_id)
```

Comment:
won't this completely error out in the non superuser case?

Reply:
yeah, i need to fix this and get the non superuser's org